code/__defines/misc.dm

```diff
@@ -312,4 +312,6 @@ block( \
 #define GET_ABOVE(A) (HAS_ABOVE(A:z) ? get_step(A, UP) : null)
 #define GET_BELOW(A) (HAS_BELOW(A:z) ? get_step(A, DOWN) : null)
 
-#define isopenturf(T) istype(T, /turf/simulated/open)
\ No newline at end of file
+#define isopenturf(T) istype(T, /turf/simulated/open)
+
+#define NEW_PLAYER_PANEL "NewPlayerPanel"
```

Comment:
Nitpicking, but this define name doesn't indicate what it actually stores; maybe change to `NEW_PLAYER_PANEL_ID` or something.

Reply:
No, I think that's a fair observation. Self documenting code and all that. Especially without a comment or anything.

Reply:
Done.